strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/ArtifactManagementService.java

```diff
@@ -16,10 +16,10 @@
 public interface ArtifactManagementService extends ConfigurationService
 {
 
-    void store(String storageId,
-               String repositoryId,
-               String path,
-               InputStream is)
+    void validateAndStore(String storageId,
```

Comment:
I don't like this renaming very much. If we need to have two separate methods for this, then okay (one for validating and one for storing)... But actually, I think that the validation should be part of this method, so it should still be called `store`. Perhaps I'm not following your reasoning...? What are your thoughts?